app/src/test/java/com/weno/problem/ProblemTest.java

```diff
@@ -11,12 +11,11 @@
 
     @BeforeEach
     void setUp(){
-        problem = new Problem("dummy-test-title");
+        problem = new Problem(1L,"dummy-test-title");
```

Comment:
```suggestion
        problem = new Problem(1L, "dummy-test-title");
```

띄어쓰기로 구분해주면 좋을 것 같습니다